input/fsh/EX_Example1.fsh

```diff
@@ -9,7 +9,7 @@ Description: "mCODE Example for Primary Cancer Condition"
 * code = SCT#254637007 "Non-small cell lung cancer (disorder)"
 * extension[histologyMorphologyBehavior].valueCodeableConcept = SCT#35917007 "Adenocarcinoma"
 * bodySite = SCT#39607008 "Lung structure (body structure)"
-* bodySite.extension[laterality].valueCodeableConcept = SCT#7771000 "Left (qualifier value)"
+* bodySite.extension[locationQualifier].valueCodeableConcept = SCT#7771000 "Left (qualifier value)"
```

Comment:
This is a breaking change. Is there a clear rationale why we want to do this other than a name preference?

Reply:
approved with noted comments that probably need revisiting in subsequent updates.